requests.md

```diff
@@ -0,0 +1,25 @@
+## Request Behavior
+
+This document describes how the framework should handle incoming requests at a high level.
+
+### Disabled Routes
+
+If the route is disabled in configuration, the request should be passed on to the next middleware component or underlying web framework handler (if applicable). Alternatively, the integration can respond with `404 Not Found`.
+
+### Supported Verbs
+
+If the route is enabled in configuration, but the request verb is not supported, the integration should respond with `405 Method Not Allowed`.
+
+If the route is enabled in configuration, and the request verb is supported, the request should be handled normally as described by the appropriate section of the spec.
+
+### `Accept` Header
```

Comment:
I want to make sure I described the logic correctly per our previous discussions. @lhazlewood does this need any more details about Content-Type negotiation?